src/meta/assessment/table.ts

```diff
@@ -35,6 +35,7 @@ export interface TableProps {
   dataExport: boolean
   columnsExport?: TableColumnNames
   columnsExportAlways?: TableColumnNames
+  dataSourceVariables?: { include: Array<string>; exclude: Array<string> }
```

Comment:
wouldn't be better to add such metadata to descriptions directly ?

Reply:
Added here because we use data.rows for the variables:

https://github.com/openforis/fra-platform/pull/2437/files#diff-b27a368caf1741b55cd317792c4f8550161efbec4364eb6d2a5b48a2f6c99e74R27-R30

But it is interesting yes, I will see about this

Reply:
Yes, but i think it’s a property of data sources description

Reply:
Yes, I agree

Updated